Jenkinsfile

```diff
@@ -379,14 +379,14 @@ try {
         // trigger macos build if we're in open-source repo
         if (env.JOB_NAME == 'IDE/open-source-pipeline/master') {
           trigger_external_build('IDE/macos-v1.4')
-          trigger_external_build('IDE/qa-opensource-automation')
         }
 
         else if (env.JOB_NAME == 'IDE/open-source-pipeline/v1.3') {
           trigger_external_build('IDE/macos-v1.3')
           trigger_external_build('IDE/windows-v1.3')
         }
         parallel parallel_containers
+        trigger_external_build('IDE/qa-opensource-automation')
```

Comment:
We should only do this if we're building `open-source-pipeline/master` (as otherwise service branch builds will also needlessly trigger an automation run against master)